ci/zkasm-result.py

```diff
@@ -1,11 +1,21 @@
 import os
 import csv
 import sys
+import argparse
 
 
-tests_dir = 'cranelift/zkasm_data/spectest/i64'
-generated_dir = 'cranelift/zkasm_data/spectest/i64/generated'
-state_csv_path = 'cranelift/codegen/src/isa/zkasm/docs/state.csv'
+parser = argparse.ArgumentParser(description='Example script to demonstrate flag usage.')
```

Comment:
Nice!

Reply:
Yah, description from example looks quite strange (= Anyway, highly unlikely we will have multiple argparsers and need meaningful descriptions in this script, so...